NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs

```diff
@@ -24,22 +23,21 @@ public override void Process(PDAEntryProgress packet)
             using (packetSender.Suppress<PDAEntryProgress>())
             {
                 TechType techType = packet.TechType.ToUnity();
-                PDAScanner.EntryData entryData = PDAScanner.GetEntryData(techType);
 
                 PDAScanner.Entry entry;
                 if (PDAScanner.GetPartialEntryByKey(techType, out entry))
                 {
                     if (packet.Unlocked > entry.unlocked)
                     {
-                        Log.Info("PDAEntryProgress Upldate Old:" + entry.unlocked + " New" + packet.Unlocked);
+                        Log.Info($"PDAEntryProgress Update For TechType:{techType} Old:{entry.unlocked} New:{packet.Unlocked}");
                         entry.unlocked = packet.Unlocked;
                     }
                 }
                 else
                 {
-                    Log.Info("PDAEntryProgress New TechType:" + techType + " Unlocked:" + packet.Unlocked);
+                    Log.Info($"PDAEntryProgress New TechType:{techType} Unlocked:{packet.Unlocked}");
                     MethodInfo methodAdd = typeof(PDAScanner).GetMethod("Add", BindingFlags.NonPublic | BindingFlags.Static, null, new Type[] { typeof(TechType), typeof(int) }, null);
```

Comment:
We might wanna use the Reflection helper class we have

Reply:
Doesn't look like we are able to do that as c# doesnt allow static types (which PDAScanner is) to be used as type arguments which is what our helper uses.

Reply:
But you could put the MethodInfo outside the method as a readonly field. Would improve performance a bit.